.eslintrc.json

```diff
@@ -16,6 +16,7 @@
     "node": false
   },
   "rules": {
+    "react/prop-types": "off",
```

Comment:
We don't have to check this in, I'm just turning it off for new because using `React.memo` inline breaks this rule for some reason.